prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -468,4 +468,31 @@ internal class ElementTests {
             assertThat(this.truncateIfNeeded("abcde")).isEqualTo("")
         }
     }
+
```

Comment:
Why is this change in here?

Reply:
my question was actually: "Why is the PR that adds ktlint also adding an unrelated unit test?"

Reply:
I have the same question. This is a new test, and does not appear to be related to linting.

Reply:
Test case belongs to another branch. I'm deleting it from this one.